aiomcache/pool.py

```diff
@@ -39,29 +37,20 @@ def acquire(self):
 
         :return: ``tuple`` (reader, writer)
         """
-        while self._size < self._minsize:
+        while self.size() == 0 or self.size() < self._minsize:
             _conn = yield from self._create_new_conn()
-
-            # Could not create new connection
             if _conn is None:
                 break
-            yield from self._pool.put(_conn)
+            self._pool.put_nowait(_conn)
```

Comment:
Yeps, my mistake I can see that the `_create_new_connection` already takes care of that kind of situations.